Library/Formula/log4cxx.rb

```diff
@@ -37,25 +39,137 @@ def install
   end
 end
 __END__
+diff --git a/src/main/cpp/locationinfo.cpp b/src/main/cpp/locationinfo.cpp
+index e76ea29..2e1950c 100644
+--- a/src/main/cpp/locationinfo.cpp
++++ b/src/main/cpp/locationinfo.cpp
```

Comment:
Can you do something to separate out the new patches from the ones already in the formula? Hosting the new patch outside the formula in a gist should be okay. Has this been submitted to the log4cxx project / has it already been committed?